worker/worker.go

```diff
@@ -303,3 +308,24 @@ func SetBinaryChecksum(checksum string) {
 func InterruptCh() <-chan interface{} {
 	return internal.InterruptCh()
 }
+
+// NewPollerBehaviorSimpleMaximum creates a PollerBehavior that allows the worker to start up to a maximum number of pollers.
+//
+// NOTE: Experimental
+func NewPollerBehaviorSimpleMaximum(
```

Comment:
Open question if these should take options instead

Reply:
Don't think so. Worst case scenario if we ever need to add anything it can be a new behavior.